ci/measure.py

```diff
@@ -37,151 +35,41 @@ def delete_dir_contents(dir):
-                    "dataset_root": "/home/xilinx/datasets",  # TODO: env var
-                    "batchsize": 100,
-                    "platform": "zynq-iodma",
-                    "runtime": 30,  # only relevant for idle baseline run
-                    "frequency": 100.0,  # will be overwritten by settingsfile (TODO)
-                    "device": 0,  # TODO: unnecessary?
-                }
-            else:
-                driver_file = f"{extract_dir}/driver/driver.py"
-                driver_args = {
-                    "settingsfile": f"{extract_dir}/driver/settings.json",
-                    "reportfile": f"{extract_dir}/measured_performance.json",
-                    "runtime": 30,  # not relevant for live FIFO-Sizing
-                    "frequency": 100.0,  # will be overwritten by settingsfile (TODO)
-                    "seed": 1,
-                    "device": 0,
-                }
-
-            # Generate (static) measurement config
-            # TODO: make this configurable (e.g., to run accelerator at multiple frequencies)
-            # TODO: make base configuration board-specific
-            measurement_cfg = {
-                "experimentDesc": "Automatically triggered flow",
-                "global": {
-                    "FINN": {
-                        "driver": f"{extract_dir}/driver",
-                        "bitstream": f"{extract_dir}/bitfile/finn-accel.bit",
-                    },
-                    "PAF": {
-                        "rails": [
-                            "0V85",
-                            "1V2_PL",
-                            "1V8",
-                            "3V3",
-                            "2V5_DC",
-                            "1V2_PS",
-                            "1V1_DC",
-                            "3V5_DC",
-                            "5V0_DC",
-                        ],
-                        "sensors": [],
-                        "board": "rfsoc2x2",
-                    },
-                    "experiment_path": driver_file,
-                    "import_paths": [],
-                    "report_path": f"{extract_dir}",
-                },
-                "experiments": [
-                    {
-                        "title": "idle",
-                        "functions": [{"name": "run_idle", "args": [], "kwargs": driver_args}],
-                        "num_runs": 1,
-                        "warmup": 10,
-                    },
-                    {
-                        "title": "load",
-                        "functions": [{"name": "main", "args": [], "kwargs": driver_args}],
-                        "num_runs": 1,
-                        "warmup": 10,
-                    },
-                ],
-            }
-            with open(f"{extract_dir}/measurement_config.json", "w") as f:
-                json.dump(measurement_cfg, f, indent=2)
+            print(f"extract_dir: {extract_dir}")
```

Comment:
This debug print is useless since it will always print "extract_dir: measurement"